oauth2.go

```diff
@@ -55,6 +55,15 @@ type Application struct {
 	Owner               *User     `json:"owner"`
 	Bot                 *User     `json:"bot"`
 	Team                *Team     `json:"team"`
+	VerifyKey           string    `json:"verify_key"`
+	RPCOrigins          []string  `json:"rpc_origins,omitempty"`
+	TeamsOfServiceURL   string    `json:"teams_of_service_url,omitempty"`
+	PrivacyPolicyURL    string    `json:"privacy_policy_url,omitempty"`
+	Summary             string    `json:"summary"`
+	GuildID             string    `json:"guild_id,omitempty"`
+	PrimarySkuID        string    `json:"primary_sku_id,omitempty"`
```

Comment:
A bit of a nit, but see: https://github.com/golang/go/wiki/CodeReviewComments#initialisms
> Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". As an example: ServeHTTP not ServeHttp. For identifiers with multiple initialized "words", use for example "xmlHTTPRequest" or "XMLHTTPRequest".

Although it looks a bit weird, `PrimarySKUID` is the way to go here. I know it's small, but given that all this change does is add fields, we might as well name them right. 🙂